### DISCUSSION TITLE
感谢大佬开源

### DISCUSSION THREAD

**Discussion by HolyBor** [ZelKnow/Hearthbot#6] [General]
听说大佬开源之后，我就下载代码库想尝试造一个炉石bot，虽然我没成功，但我一个群友成功了，现在群里又能用上bot了（
bot的QQ空间里说是被频繁举报，举报者总不可能会和机器人有仇吧，听说腾讯为了推广自己的Q群管家，一直在打压QQ群机器人，会和这件事有联系吗？
无论怎样，十分感谢大佬开源，给我们带来这么方便的bot

**Comment by ZelKnow** [comment]
用bot的人多了就会出现这种情况，所以最好就是在小范围内使用，比较安全

**Comment by qqwwee121** [comment]
感谢大佬开源
